### PR TITLE
ENH: allow users to specify odo kwargs in from_blaze

### PR DESCRIPTION
**Pull request by llllllllll**

### CODE REVIEW

File: zipline/pipeline/loaders/blaze/core.py [hunk 1]
```diff
@@ -221,20 +203,25 @@ class ExprData(namedtuple('ExprData', 'expr deltas resources')):
...
     def __repr__(self):
         # If the expressions have _resources() then the repr will
-        # drive computation so we box them.
+        # drive computation so we take the str here.
         cls = type(self)
         return super(ExprData, cls).__repr__(cls(
-            _ExprRepr(self.expr),
-            _ExprRepr(self.deltas),
-            self.resources,
+            str(self.expr),
```
Comment (richafrank) [inline zipline/pipeline/loaders/blaze/core.py]: Why the `str` change?

Comment (llllllllll) [inline zipline/pipeline/loaders/blaze/core.py]: the old box was the same as this but more complicated.

File: zipline/pipeline/loaders/blaze/core.py [hunk 2]
```diff
@@ -595,9 +585,11 @@ def from_blaze(expr,
     ds = new_dataset(dataset_expr, deltas)
     # Register our new dataset with the loader.
     (loader if loader is not None else global_loader)[ds] = ExprData(
-        dataset_expr,
-        deltas,
-        resources,
```
Comment (richafrank) [inline zipline/pipeline/loaders/blaze/core.py]: Why doesn't ExprData need the resources anymore?

Comment (llllllllll) [inline zipline/pipeline/loaders/blaze/core.py]: we are binding the resources into the expression like we do with the earnings loader

Comment (richafrank) [inline zipline/pipeline/loaders/blaze/core.py]: Ok, I follow now.

**Pull request merged**